s/spring-boot/springboot_ubi_8.5.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Package       : spring-boot
+# Version       : 3.0.1
+# Source repo   : https://github.com/spring-projects/spring-boot
+# Tested on     : ubi8.5
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Chandranana Naik <Chandranana.Naik@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+VERSION=v3.0.1
```

Comment:
Please make this version as configurable.
`VERSION=${1:-v3.0.1}`

Reply:
Done